front/src/modules/trainschedule/components/ManageTrainSchedule/Map.tsx

```diff
@@ -77,20 +77,20 @@ const Map = () => {
     bottom: 20,
   };
 
-  const { getFeatureInfoClick } = useOsrdConfSelectors();
-  const featureInfoClick = useSelector(getFeatureInfoClick);
+  // const { getFeatureInfoClick } = useOsrdConfSelectors();
+  // const featureInfoClick = useSelector(getFeatureInfoClick);
   const { updateFeatureInfoClick } = useOsrdConfActions();
 
-  const closeFeatureInfoClickPopup = useCallback(() => {
-    if (featureInfoClick.displayPopup) {
-      dispatch(
-        updateFeatureInfoClick({
-          displayPopup: false,
-          feature: undefined,
-        })
-      );
-    }
-  }, [featureInfoClick]);
+  // const closeFeatureInfoClickPopup = useCallback(() => {
+  //   if (featureInfoClick.displayPopup) {
+  //     dispatch(
+  //       updateFeatureInfoClick({
+  //         displayPopup: false,
+  //         feature: undefined,
+  //       })
+  //     );
+  //   }
+  // }, [featureInfoClick]);
```

Comment:
Remove comments if not used. We don't need this anymore ?

Reply:
Tested it and we need to keep this feature in case we clicked on the map and want to open the search modal for example. It prevents the overlap of both the popover and the modal.